_posts/simulation-machine/individual-claims-history-simulation-machine.Rmd

```diff
@@ -0,0 +1,134 @@
+---
+title: "simulationmachine: Synthesizing Individual Claims Data"
+description: |
+  Generating claims data that can be used for loss reserving research and backtesting.
+authors:
+  - name: "Andrea Gabrielli"
+    url: https://math.ethz.ch/utils/search.html?persid=173657&pagetype=people
+    affiliation: RiskLab, ETH Zürich
+    affiliation_url: https://risklab.ch/
+  - name: "Kevin Kuo"
+    url: https://kevinykuo.com/
+    affiliation: Kasa AI
+    affiliation_url: https://kasa.ai/
+  - name: "Mario V. Wüthrich"
+    url: https://people.math.ethz.ch/~wueth/
+    affiliation: RiskLab, ETH Zürich
+    affiliation_url: https://risklab.ch/
+slug: simulationmachine
+bibliography: bibliography.bib
+output: distill::distill_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Today we're excited to announce the [simulationmachine](https://github.com/kasaai/simulationmachine) package, based on the paper *An Individual Claims History Simulation Machine*[@gabrielli2018individual]. We created this package for the practitioner or researcher who needs realistic data for testing reserving or claims analytics modeling workflows.
+
+The package allows you to simulate individual claim characteristics along with their cash flow histories. We first show a quick example before delving into details of the algorithm.
+
+## simulationmachine
+
+You can download simulationmachine from GitHub with
+
+```r
+# install.packages("remotes")
+remotes::install_github("kasaai/simulationmachine")
+```
+
+First, specify the desired characteristics of the simulation:
```

Comment:
I think a short paragraph and link to the conjure package would increase clarity here because I think readers might get hung up on the variable "charm". It makes a lot more sense if one understands the metaphor used in the conjure package.